wicketstuff-pdf-js-parent/wicketstuff-pdfjs-examples/src/main/java/org/wicketstuff/pdfjs/example/PdfJsDemoPage.html

```diff
@@ -34,16 +50,40 @@
 				$('#zoomOut').click(function () {
 					Wicket.Event.publish(WicketStuff.PDFJS.Topic.ZOOM_OUT, {"canvasId": canvasId});
 				});
-				 $('#page_num').change(function () {
+				$('#zoom').change(function () {
+					var scale = $(this).val();
+					Wicket.Event.publish(WicketStuff.PDFJS.Topic.ZOOM_TO, {"canvasId": canvasId, "scale" : scale});
+				});
+				$('#page_num').change(function () {
 					var page = parseInt($(this).val());
 					Wicket.Event.publish(WicketStuff.PDFJS.Topic.GOTO_PAGE, {"canvasId": canvasId, "page" : page});
 				});
+				$('#print').click(function () {
+					Wicket.Event.publish(WicketStuff.PDFJS.Topic.PRINT, {"canvasId": canvasId});
+				});
                 Wicket.Event.subscribe(WicketStuff.PDFJS.Topic.CURRENT_PAGE, function (jqEvent, pageNumber, data) {
                     $('#'+data.canvasId).closest('.pdfPanel').siblings().find('#page_num').val(pageNumber);
                 });
 				Wicket.Event.subscribe(WicketStuff.PDFJS.Topic.TOTAL_PAGES, function (jqEvent, total, data) {
 					$('#'+data.canvasId).closest('.pdfPanel').siblings().find('#page_count').text(total);
 				});
+				Wicket.Event.subscribe(WicketStuff.PDFJS.Topic.CURRENT_ZOOM, function (jqEvent, zoom, data) {
+
+				    var zoomDropDown =  $('#'+data.canvasId).closest('.pdfPanel').siblings().find('#zoom');
+
+				    $("option.custom").each(function() {
```

Comment:
Maybe it would be better to visit all options and:
1) remove the `.custom` one
2) deselect any selected one if `zoom` is truthy.